generator.yml

```diff
@@ -71,6 +86,8 @@ configs:
   - &arm32_allmod      {config: allmodconfig,                                                                                              << : *arm-triple,           << : *kernel_modules}
   - &arm32_allno       {config: allnoconfig,                                                                                               << : *arm-triple,           << : *kernel_modules}
   - &arm32_allyes      {config: allyesconfig,                                                                                              << : *arm-triple,           << : *kernel_modules}
+  - &arm32_fedora      {config: [*arm32-fedora-config-url, CONFIG_BPF_PRELOAD=n],                                                          << : *arm-triple,           << : *kernel_modules}
```

Comment:
can you add a comment with link to why BPF_PRELOAD is disabled?

Reply:
ah, it's below. add it here, too?

Reply:
Sure.